offchain-modules/bridge-ckb-sdk/src/cell_collector.rs

```diff
@@ -0,0 +1,158 @@
+use crate::indexer::{Cell, IndexerRpcClient, Order, Pagination, ScriptType, SearchKey};
```

Comment:
The `bridge` in path is redundant, when we use the crate, we refer it by package name, so `sdk` is enough to replace `bridge-ckb-sdk`.

An example is this:

https://github.com/nervosnetwork/ckb/blob/develop/traits/Cargo.toml

the crate in `traits` dir is named `ckb-traits`.